### ISSUE TITLE
'DefaultGateway' is obsolete.  

### ISSUE THREAD

**Issue by krisjturn** [Microsoft/SDN#128]
Trying to run the script.  I am able to get the network controllers to deploy now.  Now I am getting errors when the script tries to create the HNV and Transit networks.

Warning Parameter 'DefaultGateway' is obsolete.  DefaultGatway is deprecated.  Use NetworkRoute.

I am guess this is within the script itself and that it is not using the proper cmdlts.  Is there an updated script  around?

**Comment by grcusanz** [comment]
HI Krisjturn, this is just a warning that you can safely ignore.   Are there any additional error messages that you were receiving?

**Comment by krisjturn** [comment]
The script actually fails when I get the above message.  It is at the point where it is trying to create the IP Pool for the HNV network.  I manually created the networks and re-ran the script.  Everything seemed to work at this point.

**Comment by krisjturn** [comment]
This isn't related but I am having difficulty with L3 Forwarding.  There isn't much documentation outside of a Microsoft Doc and a blog on it.

**Comment by grcusanz** [comment]
Hi Kris, are you managing this environment using SC VMM (I assume you're using VMM SDN Express), or are you managing it through PowerShell without VMM (you should be using the regular SDN Express then)?

**Comment by krisjturn** [comment]
VMM.  Yep, the latest VMM SDN Express script.  Well as of about 3 weeks ago.

**Comment by grcusanz** [comment]
[Here](https://docs.microsoft.com/en-us/system-center/vmm/sdn-route-network-traffic) is the official Docs link for setting up L3 forwarding through SC VMM.  [This blog](https://blogs.technet.microsoft.com/larryexchange/2016/06/01/step-by-step-for-deploying-a-sdnv2-using-vmm-part-4/) is the best walkthrough showing how to setup L3 forwarding.  Are these the docs you've already found?   Is there a specific area regarding L3 forwarding I can help clarify?

**Comment by grcusanz** [comment]
Hi Kris,  are you still having issues with L3 gateways we can help with?  If not I will close this issue.  Thanks!

**Comment by krisjturn** [comment]
We ran into some issues with configuration of the L3 Gateway.  We couldn't get BGP configured so we went with static routes within the L3 Gateway.  Which is OK for now but as we grow and start adding more L3 Gateways we would like to revisit that.  Now, what I am doing today since I have some free cycles is patching my SDN environment and will be creating some test tenant networks to test L3 gateways.  I will admit, my VHDX that I used was a few months old and wasn't updated in a while.  (Shh, don't tell anyone.)   :)
